creator/ingest_runs/data_generator/ingest_package/extract_configs/genomic.py

```diff
@@ -5,24 +5,20 @@
 https://kids-first.github.io/kf-lib-data-ingest/tutorial/extract.html for
 information on writing extract config files.
 """
-import pandas as pd
-
-from kf_lib_data_ingest.common import constants  # noqa F401
+import os
+from kf_lib_data_ingest.common import constants, pandas_utils  # noqa F401
 from kf_lib_data_ingest.common.concept_schema import CONCEPT
 from kf_lib_data_ingest.etl.extract.operations import *
+from kf_lib_data_ingest.common.io import read_df
 
-# TODO - Replace this with a URL to your own data file
```

Comment:
Instead of incorporating the file metadata this way, we can do this separately in another extract config. We just use the standard s3 scrape extract config to extract the s3 file metadata and then merge it into our df in the transform module.